utils/api.js

```diff
@@ -38,4 +38,21 @@ export const getTopics =()=> {
     })
   }
   
+  export const patchVotes=(article_id)=>{
+    return newsApi.patch(`articles/${article_id}`,{inc_votes:1}).then((result)=>{
+        return result.data
+    })
+  }
+  
+  export const patchVotesDown=(article_id)=>{
+    return newsApi.patch(`articles/${article_id}`,{inc_votes:-1}).then((result)=>{
+        return result.data
+    })
+  }
```

Comment:
two api calls wouldn't be needed, you could put either a 1 or -1 into the function based on wether its an upvote or a downvote